modules/custom/d_p_tiles/d_p_tiles.module

```diff
@@ -15,7 +15,7 @@ function d_p_tiles_theme($existing, $type, $theme, $path) {
     'paragraph__d_p_tiles' => [
       'base hook' => 'paragraph',
     ],
-    'field__paragraph__field_d_image__d_p_tiles' => [
+    'field__paragraph__field_d_media_image__d_p_tiles' => [
```

Comment:
To musi być osobny theme czy łapałoby się pod theme suggestions?

Reply:
Ziom! wychodzi na to, że musi.
po usunięciu tych linijek z hooka _theme
mamy co prawda podpowiedzi: 
![image](https://user-images.githubusercontent.com/37533838/70530995-570e9e00-1b54-11ea-909d-8e31ad8e95cd.png)
ale nie widzi tych templates w module: 
![image](https://user-images.githubusercontent.com/37533838/70531018-62fa6000-1b54-11ea-90ca-157373097012.png)